lib/generators/rspec/scaffold/templates/index_spec.rb

```diff
@@ -18,9 +18,10 @@
 
   it "renders a list of <%= ns_table_name %>" do
     render
-    cell_selector = Rails::VERSION::STRING >= '7' ? 'div>p' : 'tr>td'
+
+<% cell_selector = Rails::VERSION::STRING.to_f >= 7.0 ? 'div>p' : 'tr>td' -%>
```

Comment:
I was wondering if we need the parent Rails version
 
```suggestion
<% cell_selector = ::Rails::VERSION::STRING.to_f >= 7.0 ? 'div>p' : 'tr>td' -%>
```